code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -198,7 +199,8 @@
 		new V(podman)
 	podman.hardset_dna(null,null,null,podman.real_name,blood_type, new /datum/species/pod,features)//Discard SE's and UI's, podman cloning is inaccurate, and always make them a podman
 	podman.set_cloned_appearance()
+	var/datum/species/species = podman.dna.species
+	species.exotic_blood = max(reagents_add)
```

Comment:
what if it has no reagents

Reply:
@ATH1909 said it wouldnt matter, I hadnt considered that. ill readd the 1% water production, then?

Reply:
If reagents add is null or empty, the podperson will be given a normal blood type.